### PR TITLE
OZ-548: Added relevant self documenting comments to the .env file.

### PR DESCRIPTION
**Pull request by kdaud**
Ticket → [OZ-548](https://mekomsolutions.atlassian.net/browse/OZ-548)

[OZ-548]: https://mekomsolutions.atlassian.net/browse/OZ-548?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Ruhanga — APPROVED** [review]
Generally LGTM.

File: .env [hunk 1]
```diff
@@ -1,28 +1,66 @@
...
+# OpenMRS
 O3_URL_DEV=https://ozone-dev.mekomsolutions.net
 O3_URL_QA=https://ozone-qa.mekomsolutions.net
-ODOO_URL_DEMO=https://erp.demo.ozone-his.com
+O3_URL_DEMO=https://demo.ozone-his.com
+
+# Superset
+ANALYTICS_URL_DEV=https://analytics.ozone-dev.mekomsolutions.net
+ANALYTICS_URL_QA=https://analytics.ozone-qa.mekomsolutions.net
+ANALYTICS_URL_DEMO=https://analytics.demo.ozone-his.com
```
Comment (mks-d) [inline .env:26]: `ANALYTICS_URL_` → `SUPERSET_URL_`

File: .env [hunk 2]
```diff
@@ -1,28 +1,66 @@
...
+# This file is loaded each time the automated E2E test suite is run.
+# It sets a bunch of envvars that drive the behaviour of the automated E2E test suite.
+#
+
+# The target environment that will be tested.
+# Possibles values are dev, qa, demo.
+TEST_ENVIRONMENT=dev
+
+# Set to true to test Ozone Pro, false to test Ozone FOSS
+RUNNING_OZONE_ON_PRO=true
```
Comment (mks-d) [inline .env:11]: `RUNNING_OZONE_ON_PRO` → `TEST_PRO`

File: .env [hunk 3]
```diff
@@ -1,28 +1,66 @@
...
 ODOO_USERNAME_ON_FOSS=admin
 ODOO_PASSWORD_ON_FOSS=admin
+
+# SENAITE test user credentials for FOSS
 SENAITE_USERNAME_ON_FOSS=admin
 SENAITE_PASSWORD_ON_FOSS=password
+
+# (Pro only) Ozone test user credentials
+O3_USERNAME=jdoe
+O3_PASSWORD=password
```
Comment (mks-d) [inline .env:62]: `O3_` → `OZONE_`

**Review by mks-d — CHANGES_REQUESTED** [review]

**Pull request merged**